### DISCUSSION TITLE
Lighthouse differences on Chrome Windows vs Chrome Mac

### DISCUSSION THREAD

**Discussion by learning101010** [GoogleChrome/lighthouse#13208] [Q&A]
Good Day Everyone

I test my website today both Windows Chrome and Mac Chrome , but the result is not the same 
This laptop is connected on same network with speed of 200mbps, 

Mac Spec / i5 1,6 / 8gb of ram
Windows / i7 9th / 12gb or ram

I test my website in incognito mode
On Windows i get : 92% / 88% / 90% (I close and open the chrome) Latest Version of Chrome
But on Mac i get : 76%/54%/80% (I close and open the chrome) Latest Version of Chrome

Note this website is not yet available for public , i can only access this with my IP.

Looking forward for your suggestion on what happen why i get not near similar result on windows.

**Comment by brendankenny** [comment]
Without a URL to test there's not really any way to get into specifics of this situation. You might find it helpful to look over the variability docs: https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md

Keep in mind that the machine you run the tests on isn't the only source of variance, there's also differences in the network at that moment, the server, and the content of the site itself, particularly any dynamic parts of the page.
